config/ncit.iris

```diff
@@ -44,6 +44,11 @@
 +(http://purl.obolibrary.org/obo/OBI_0000470):http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#C48040 Matrix-Assisted Laser Desorption/Ionization
 +D:(http://purl.obolibrary.org/obo/OBI_0000070):http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#C80474 Parameters
 +D:(http://purl.bioontology.org/ontology/npo#NPO_1416):http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#C62321 Asymmetrical field flow fractionation
++D:(http://purl.bioontology.org/ontology/npo#NPO_1945):http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#C64657 Dilution
```

Comment:
D (for including down tree) can be removed as there is no down tree for Dilution in NCIT.

Added